props/saber_fett263_buttons.h

```diff
@@ -687,103 +689,20 @@ class SoundLibrary {
   void SaySwingColor() { Play("mswing.wav"); }
   void SaySwingIgnition() { Play("mswingon.wav"); }
   void SaySwingOnSpeed() { Play("mswingsp.wav"); }
-  void SayThrustIgnition() { Play("mthrston..wav"); }
+  void SayThrustIgnition() { Play("mthrston.wav"); }
   void SayTwistIgnition() { Play("mtwiston.wav"); }
   void SayTwistRetraction() { Play("mtwstoff.wav"); }
-  void SayUp() { Play("mup.wav"); }
+  void SayUp() { Play("mup.wav"); } // Sound for increase
   void SayVolumeMenuEnd() { Play("vmend.wav"); }
-  void SayVolumeUp() { Play("volup.wav"); }
-  void SayZoom() { Play("mzoom.wav"); } // Zoom in for color menu "zooming in"
-  void SayVolumeDown() { Play("voldown.wav"); }
+  void SayVolumeUp() { Play("volup.wav"); } // Sound for increasing volume
+  void SayZoom() { Play("mzoom.wav"); } // Sound for color menu "zooming in"
+  void SayVolumeDown() { Play("voldown.wav"); } // Sound for decreasing volume
   void SayGestureMenu() { Play("mgestsub.wav"); }
   void SaySettingsMenu() { Play("msetsub.wav"); }
   void SayStyleSettings() { Play("stylstm.wav"); }
 
   void SayColor(int n) {
```

Comment:
replace int with "SayColorName" (hopefully it won't lead to a bunch of errors)

Reply:
SayColorName was removed, we're using SFX_clrlst.Select() instead.

Reply:
Right, it's called ColorName now.
Replace "int" with "ColorName".

Reply:
ok, updated